### PR TITLE
junit: Add maven and gradle classes to custom hook excludes

### PR DESCRIPTION
**Pull request by florianGla**

### CODE REVIEW

File: src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java [hunk 1]
```diff
@@ -52,6 +52,8 @@ static void forFuzzing(ExtensionContext executionRequest) {
     String filter = String.join(File.pathSeparator, instrumentationFilter.split(","));
     System.setProperty("jazzer.custom_hook_includes", filter);
     System.setProperty("jazzer.instrumentation_includes", filter);
+    System.setProperty("jazzer.instrumentation_excludes",
+        String.join(File.pathSeparator, "org.apache.maven.**, org.gradle.**"));
```
Comment (fmeum) [inline src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java:56]: These classes *should* not be matched by the default instrumentation filter we compute in line 51. Do the projects with which you are experiencing issues have `jazzer.instrument` set?

Comment (fmeum) [inline src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java:56]: Could you move these exclusions to the list below in `applyCommonConfiguration` and check whether that already fixes the issue?

Comment (florianGla) [reply src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java]: I don't think so.

File: src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java [hunk 2]
```diff
@@ -60,6 +60,6 @@ private static void applyCommonConfiguration() {
         String.join(File.pathSeparator, "com.google.testing.junit.**", "com.intellij.**",
             "io.github.classgraph.**", "junit.framework.**", "net.bytebuddy.**",
             "org.apiguardian.**", "org.assertj.core.**", "org.hamcrest.**", "org.junit.**",
-            "org.opentest4j.**"));
+            "org.opentest4j.**", "org.apache.maven.**, org.gradle.**"));
```
Comment (fmeum) [inline src/main/java/com/code_intelligence/jazzer/junit/AgentConfigurator.java:63]:
```suggestion
            "org.opentest4j.**", "org.apache.maven.**", "org.gradle.**"));
```

**Review by fmeum — APPROVED** [review]

**Pull request merged**